scripts/shared/lib/deploy_funcs

```diff
@@ -122,3 +122,17 @@ function load_deploytool() {
     . "$deploy_lib"
 }
 
+function find_submariner_namespace() {
+    kubectl get pods -A | awk '/submariner/{ print $1 }' | grep -v broker | head -n 1
+}
+
+function reload_pods() {
+    local resource_type=$1 # the resource type can be deployment or daemonset
+    local resource_name=$2 # the name of the resource
+    local namespace
+    namespace="$(find_submariner_namespace)"
```

Comment:
If the reload_pods is run on a non-submariner deployment, the error message might be hard to read.
Can you add a check to ensure that namespace is not empty and log an appropriate error message.

Reply:
This can only be used in our kind deployments mostly, but I'll add it just in case.

Reply:
Done :-)